archivers/par2/Portfile

```diff
@@ -1,9 +1,10 @@
 PortSystem        1.0
 PortGroup         compiler_blacklist_versions 1.0
+PortGroup         github 1.0
 
 name              par2
-version           0.4
-revision          4
+github.setup      Parchive par2cmdline v0.8.1
```

Comment:
the "v" should not be part of the version, it should be "0.8.1 v" instead

Reply:
Ah, that is what I missed. Thx! Done.